src/Model/Behavior/DuplicatableBehavior.php

```diff
@@ -266,8 +266,10 @@ protected function _drillDownEntity($action, EntityInterface $entity, array $par
             return;
         }
 
-        foreach ($entity->{$prop} as $key => $e) {
-            $this->_drillDownEntity($action, $e, $parts, $value);
+        if ($entity->{$prop} !== null) {
```

Comment:
`is_iterable()` check would be even better

Reply:
That would make the minimum PHP version for the plugin 7.1.0 is that okay? I don't see a PHP version constraint in the `composer.json` file, and it stipulates CakePHP 3.5 which would be PHP 5.6

Reply:
Oh right, in that case is_array() and instanceof Traversable check then.